.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```
@@ -15,6 +22,23 @@ body:
     validations:
       required: true
+  - type: textarea
+    id: version-info
+    attributes:
+      label: Version Information
+      description: |
+        If using Docker, please run and provide the output of:
+        ```bash
+        docker images | grep librechat
+        ```
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)